public/scripts/Table.js

```diff
@@ -38,6 +38,7 @@ var Table = React.createClass({
         return data.map(function(item) {
             var cells = cols.map(function(colData) {
                 return <td> {item[colData.key]} </td>;
+              {item[colData.key]}.sort(sort_by('name1', asec));
```

Comment:
- Use the sort function in a separate method/function.....
```
getSortedRows : function (originalData) {
 var sortedData = originalData.sort(SORT_FUNCTION_HERE);
 return  sortedData;
}
```

use [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) as a reference for writing a sorting function.